plugin/server/management.py

```
@@ -0,0 +1,365 @@
+                author = PluginAuthor(
+                    name=author_data.get("name"),
+                    email=author_data.get("email")
+                )
+            
+            # 解析并检查插件依赖
+            dependencies = _parse_plugin_dependencies(conf, logger, plugin_id)
+            dependency_check_failed = False
+            if dependencies:
+                logger.info("Plugin {}: found {} dependency(ies)", plugin_id, len(dependencies))
+                for dep in dependencies:
+                    # 检查依赖（包括简化格式和完整格式）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)